languages/csharp/exercises/concept/chars/.meta/Example.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Text;
+
+public static class CharUtils
+{
+    public static bool DetectInvalidCharComination(string str)
```

Comment:
In theory, a student could also solve this as follows, right?

```csharp
public static bool DetectInvalidCharComination(string str) => str.Contains("->");
```

One way we could maybe force students to work with chars is to do something like this, where the characters are provided as parameters:

```chars
public static bool DetectInvalidCharComination(string str, char first, char second)
```